contents/linux/path.tw2

```diff
@@ -25,7 +25,6 @@ To check whether this worked, go to your home directory
 and try to run `BINARY_NAME` without providing the path:
```

Comment:
This should have been changed too I think.  `cd ~` was removed which was instructional for how to get to the home directory but the user is still being told to go to their home directory.

Reply:
That was quick! Thank you so much. I've added a PR for this here: https://github.com/exercism/interactive-cli-walkthrough/pull/73